### DISCUSSION TITLE
How accessible is tiptap?

### DISCUSSION THREAD

**Discussion by cadars** [ueberdosis/tiptap#963] [General Questions]
I couldn't find anything about it in the docs or issues, and was wondering if it follows any a11y basic practices.
Thanks!

**Comment by hanspagel** [comment]
Great questions @cadars! Unfortunately, accessibility didn't play a role when developing tiptap v1. That said, from what I’ve seen it didn’t play a role in the development of other open-source editors aswell.

The only good news here is, that we plan to tackle a11y issues with tiptap v2. We’re currently in the phase of researching what we need to do (hint: it’s a lot), so it’s pretty early.

If you have any knowledge about the topic, I’d love to invite you to the private alpha and would love to get input from you! 🙌
If that’s the case, reach out to humans@tiptap.dev.

Anyway, thanks for asking!

**Reply by cadars** [reply]
Thanks for your answer. Also thanks for the invitation, but I have limited knowledge about a11y, so I'll just sponsor the project  if I want to access it :)
